md/api-extensions.md

```diff
@@ -31,6 +31,10 @@ REST API extensions use the same authorization mechanism as the standard Bonita
 **Warning**: If you are running Bonita BPM in a cluster, after you import a REST API extension, you must restart the application server on all the cluster nodes.
 :::
 
+::: warning
+**Warning**: As the deployment of the REST API extensions automatically creates the appropriate resource permission mapping, be careful not to [push a configuration](BonitaBPM_platform_setup.md#update_platform_conf) that has been updated before the push. This would overwrite the deployed configuration. To avoid doing so, pull the latest configuration, edit the needed elements and push it back to the Bonita runtime in a pretty short amount of time.
```

Comment:
"not to push a configuration **that does not contain the latest permission updates**." would be clearered I think